unstable/wlr-export-dmabuf-unstable-v1.xml

```diff
@@ -0,0 +1,230 @@
+        each component being an unsigned 32-bit value. Whole seconds are in
+        tv_sec which is a 64-bit value combined from tv_sec_hi and tv_sec_lo,
+        and the additional fractional part in tv_nsec as nanoseconds. Hence,
+        for valid timestamps tv_nsec must be in [0, 999999999].
+        The seconds part may have an arbitrary offset at start.
+      </description>
+      <arg name="tv_sec_hi" type="uint"
+           summary="high 32 bits of the seconds part of the timestamp"/>
+      <arg name="tv_sec_lo" type="uint"
+           summary="low 32 bits of the seconds part of the timestamp"/>
+      <arg name="tv_nsec" type="uint"
+           summary="nanoseconds part of the timestamp"/>
+    </event>
+    <event name="abort">
+      <description summary="indicates the frame is no longer valid">
+        If the frame is no longer valid after the "frame" event has been called,
+        this callback will be used to inform the client to scrap the frame.
+        Source is still valid for as long as the subscription function does not
+        return NULL.
+        This may get called if for instance the surface is in the process of
+        resizing.
+      </description>
+    </event>
+
+    <request name="destroy" type="destructor">
+      <description summary="delete this object, used or not">
+        Unreferences the frame, allowing it to be reused. Must be called as soon
+        as its no longer used.
+      </description>
+    </request>
+  </interface>
+
+  <interface name="zwlr_export_dmabuf_manager_v1" version="1">
+    <description summary="manager to inform clients and begin capturing">
+      This object is a manager which informs clients about capturable windows
+      and is able to create callbacks from which to begin to receive content
+      from. The "title" argument in the "surface_info" event shall be used
+      to provide a user-readable identifier such as a window title or
+      program name.
+    </description>
+
+    <event name="surface_info">
+      <description summary="get valid surfaces for capture">
+        This will be called whenever a surface that's able to be captured
+        appears.
+      </description>
+      <arg name="title" type="string"
+           summary="window title or an identifier"/>
+      <arg name="id" type="uint"
+           summary="an identifier to create a context with"/>
+    </event>
+    <event name="surface_unavailable">
+      <description summary="notifies of a surface being unavailable to capture">
+        Called if a surface becomes unavailable to capture, for example if has
+        been closed.
+      </description>
+      <arg name="id" type="uint"
+           summary="an identifier of the surface"/>
+    </event>
+
+    <request name="capture_client">
+      <description summary="subscribe to start capturing">
+        Request to start capturing from a surface with a given id.
+        If an ID becomes unavailable, a NULL will be returned.
```

Comment:
It's not possible to return NULL because Wayland is an asynchronous protocol. Instead, you can return an abort frame.